pkg/sdk/schemas.go

```diff
@@ -1,9 +1,46 @@
 package sdk
 
-// placeholder for the real implementation.
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"time"
+)
+
+type Schemas interface {
```

Comment:
what about use schema?

Reply:
Shouldn't it be consistent with other interface names ? (Alters, Databases, etc. ?)

Reply:
I didn't mean the name, but the method from docs: https://docs.snowflake.com/en/sql-reference/sql/use-schema

Reply:
Ah ok, it was already implemented. Functions like 'use ...' are in the Client struct (client.Sessions.UseSchema())

Reply:
I added Use() function as a proxy to the client.Session.UseSchema function